src/main/java/com/unbxd/SchemaManager/Dao/Dao.java

```diff
@@ -0,0 +1,18 @@
+package com.unbxd.SchemaManager.Dao;
```

Comment:
this package name makes no sense
it should be com.unbxd.schema
which all schema related stuff in it
if it is a controller keep it in a different controller package as it has nothing to do with schema and only contacts a service
dao cannot be a package, dao should be in schema package instead